backend/models/db/attachment-attempts.model.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
```

Comment:
hmm I wasn't even aware that we had a table by this name (attachment-attempts.model.js). Maybe we can also delete this since we don't use it.